testutils/build.gradle

```diff
@@ -185,7 +197,12 @@ dependencies {
     implementation "com.nimbusds:nimbus-jose-jwt:$rootProject.ext.nimbusVersion"
     implementation 'androidx.appcompat:appcompat:1.0.2'
     implementation "junit:junit:$rootProject.ext.junitVersion"
-    implementation ("org.robolectric:robolectric:$rootProject.ext.robolectricVersion")
+    implementation ("org.robolectric:robolectric:$rootProject.ext.robolectricVersion") {
+        exclude group: "org.apache.maven"
+        exclude group: "org.apache.maven.wagon"
+        exclude group: "xmlpull"
+        exclude group: "xpp3"
```

Comment:
Why do we need to exclude these? 😀

Reply:
These are causing our duplicate classes.

Reply:
Oh. I see.